actors/common/src/main/java/org/sunbird/common/ShadowUserProcessor.java

```diff
@@ -456,7 +457,26 @@ private void changeStatusToMultiMatch(List<ShadowUser>shadowUserList){
     }
 
 
+    /**
+     * This filtering will be needed to avaoid update of claimStatus to MULTIMATCH
+     * of same user while updating.
+     *
+     * @param channel
+     * @param shadowUserList
+     */
+    private List<ShadowUser> filterByChannel(String channel, List<ShadowUser> shadowUserList) {
```

Comment:
this method is giving me feel that it will return all users who belongs to provided channel , but seems it's not true.
so please correct method name.

Reply:
DONE